L1Trigger/L1TCommon/test/runHLT.py

```diff
@@ -0,0 +1,182 @@
+# Auto generated configuration file
+# using: 
+# Revision: 1.19 
+# Source: /local/reps/CMSSW/CMSSW/Configuration/Applications/python/ConfigBuilder.py,v 
+# with command line options: debug --no_exec --conditions auto:run2_mc_25ns14e33_v4 -s DIGI:pdigi_valid,L1,DIGI2RAW,RAW2DIGI --datatier GEN-SIM-DIGI-RAW-HLTDEBUG -n 10 --era Run2_25ns --eventcontent FEVTDEBUGHLT --filein filelist:step1_dasquery.log --fileout file:step2.root
+import FWCore.ParameterSet.Config as cms
+
+from Configuration.StandardSequences.Eras import eras
+
+#process = cms.Process('L1SEQS',eras.Run2_25ns)
+process = cms.Process('L1SEQS',eras.Run2_2016)
+
+process.MessageLogger = cms.Service(
+    "MessageLogger",
+    destinations = cms.untracked.vstring('l1tdebug','cerr'),
+    l1tdebug = cms.untracked.PSet(threshold = cms.untracked.string('DEBUG')),
+    cerr = cms.untracked.PSet(threshold  = cms.untracked.string('WARNING')),
+    debugModules = cms.untracked.vstring('*'))
+
+
+# import of standard configurations
+process.load('Configuration.StandardSequences.Services_cff')
+process.load('SimGeneral.HepPDTESSource.pythiapdt_cfi')
+process.load('FWCore.MessageService.MessageLogger_cfi')
+process.load('Configuration.EventContent.EventContent_cff')
+process.load('SimGeneral.MixingModule.mixNoPU_cfi')
+process.load('Configuration.StandardSequences.GeometryRecoDB_cff')
+process.load('Configuration.StandardSequences.MagneticField_cff')
+process.load('Configuration.StandardSequences.Digi_cff')
+process.load('Configuration.StandardSequences.SimL1Emulator_cff')
+process.load('Configuration.StandardSequences.DigiToRaw_cff')
+process.load('Configuration.StandardSequences.EndOfProcess_cff')
+process.load('Configuration.StandardSequences.FrontierConditions_GlobalTag_cff')
+
+#
+# LOCAL CONDITIONS NEEDED FOR RE-EMULATION OF GT
+#
+
+from L1Trigger.L1TGlobal.StableParameters_cff import *
+from L1Trigger.L1TGlobal.TriggerMenu_cff import *
+TriggerMenu.L1TriggerMenuFile = cms.string('L1Menu_Collisions2015_25nsStage1_v7_uGT.xml')
+
+#
+# BEGIN HLT UNPACKER SEQUENCE FOR STAGE 2
+#
+
+process.hltGtStage2Digis = cms.EDProducer(
+    "L1TRawToDigi",
+    Setup           = cms.string("stage2::GTSetup"),
+    FedIds          = cms.vint32( 1404 ),
+)
```

Comment:
That's fine for py configs.

But I stress once more: fillDescriptions must contain all parameters.
And all parameters must have a value, dummy or useful default

Dummy values are to be set to usefull values in explicit py configs.